models/order.js

```diff
@@ -63,23 +64,42 @@ class OrdersManager {
     return orders;
   }
 
-  static async createOrder (order) {
+  static async createOrder(order) {
     const dataArray = ordersObjectToData(order);
     const queryResponse = await db.query(
-        "INSERT INTO orders (clientUUID, orderdate, orderstatus, ordercharge, originLatitude, originLongitude, destinationLatitude, destinationLongitude, description)" + 
+      "INSERT INTO orders (clientUUID, orderdate, orderstatus, ordercharge, originLatitude, originLongitude, destinationLatitude, destinationLongitude, description)" +
         "VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9) RETURNING *;",
-        dataArray
+      dataArray
     );
     if (!queryResponse) {
-        return null;
+      return null;
     }
     return queryResponse;
   }
 
-  static async updateOrder (order) {
-
+  static async updateStatus(updatedOrder) {
```

Comment:
Related to the ordersController, I think it is best if this is an object with ID and Status and we construct the query array here.